### PR TITLE
Comment section step 1

### PR DESCRIPTION
**Pull request by AlvieH**
step 1 of the comment section completed

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,122 @@ function addRandomGreeting() {
...
+function slideToggle(elementId, duration) {
+  var element = document.getElementById(elementId);
+  if(window.getComputedStyle(element).display === 'none')
+  {
+    slideDown(elementId, duration);
+  }
+  else {
+    slideUp(elementId, duration);
+  }
+}
```
Comment (zkoogle) [inline portfolio/src/main/webapp/script.js]: New line pls.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,122 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+* Plays a specified melody
+ */
+function play(melody) {
```
Comment (zkoogle) [inline portfolio/src/main/webapp/script.js]: You are using both "function" keywords and arrow functions. Can you be consistent over your codebase? (Choose one -- maybe arrow functions with 'const'):
https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions

Comment (AlvieH) [reply portfolio/src/main/webapp/script.js]: Done!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (AlvieH) [reply portfolio/src/main/webapp/script.js]: Done!

**Review by askewc — APPROVED** [review]

**Pull request merged**